src/components/Identity/identityUtil.js

```diff
@@ -0,0 +1,61 @@
+import { AUTH_STATES } from "./constants";
```

Comment:
Should we rename this module to simply `utils.js` since it lives in `Identity` already?